chromium/browser_type.go

```diff
@@ -387,11 +387,14 @@ func execute(
 		}()
 
 		if err := cmd.Wait(); err != nil {
-			logger.Errorf("BrowserType:execute", "cmd.Wait: %v", err)
-		}
-
-		if err := stdout.Close(); err != nil {
-			logger.Errorf("BrowserType:execute", "stdout.Close: %v", err)
+			log := logger.Errorf
+			if err.Error() == "signal: killed" || err.Error() == "exit status 1" {
+				// The browser process is killed when the context is cancelled
+				// after a k6 iteration ends, so silence the log message until
+				// we can stop it gracefully. See #new-issue-number
```

Comment:
:smile:

```suggestion
				// we can stop it gracefully. See https://github.com/grafana/xk6-browser/issues/423
```

Reply:
🤦 Thanks :)

Reply:
Resolved in: b87e9711395eb5bc473212c73ee02e27479d016c

Reply:
The hash prefix is not needed, but it's fine :)

Reply:
Ah, i was avoiding "Commit suggestion" as I didn't know about the batch feature. I will use it next time, thanks